templates/settings/store-form.php

```diff
@@ -182,6 +188,53 @@
             </div>
         </div>
 
+        <?php
+        $sections = array_filter( $products_block_appearance, function( $value ) {
+            return empty( $value );
+        } );
+
+        if ( count( $sections ) > 0 ) {
+            ?>
+            <div class="dokan-form-group">
+            <label class="dokan-w3 dokan-control-label"><?php esc_html_e( 'Store page product sections', 'dokan-lite' ); ?></label>
+
+            <div class="dokan-w5 dokan-text-left">
+
+            <?php if ( empty( $products_block_appearance['hide_featured_products'] ) ) { ?>
```

Comment:
in templating always use alternate if-else statement like this -
```php
<?php if ( empty( $products_block_appearance['hide_featured_products'] ) ) : ?>
    <!-- HTML Code -->
<?php endif; ?>
```

Same is applicable for - `foreach`

Please re-factor this.